README.md

```diff
@@ -1,62 +1,181 @@
 # Exercism v3 Local Environment
 
-This provides a local environment for running Exercism on Docker. It is very much a WIP.
+Hello 👋
 
-## Getting Started
+This repository will provide you with a local setup of Exercise that you can use to develop any part of the project.
 
-Run the following line-by-line.
+It is built using [Docker](https://www.docker.com/), which you need to install, but it does not require any real Docker knowledge to use.
+
+Our aim is to get you to a working setup within 10-minutes from now (presuming you have a decent internet connection).
+It's also possible to get each piece of Exercism working on your local machine without this repository, but that will take lots more effort and work.
+
+## Basic setup
+
+The following instructions take you through getting the most basic setup working.
+We'll explain what actually happens below the hood, and how to configure things afterwards
+
+### Getting set up
+
+To start, run the following instructions line-by-line:
 
 ```bash
 # Make a directory to host all of Exercism within
 mkdir exercism
 
-# cd into it
+# Move into the new directory
 cd exercism
 
-# Clone this repo
+# Clone this repository onto your computer
 git clone git@github.com:exercism/v3-docker-compose.git
 
-# Clone any essential repos you plan to hack on
-git clone git@github.com:exercism/v3-website.git
-git clone git@github.com:exercism/tooling-orchestrator.git
-git clone git@github.com:exercism/tooling-invoker.git
-git clone git@github.com:exercism/csharp-test-runner.git
-
+# Move into the new directory
 cd v3-docker-compose
 
-# start with the default stack
+# Create your "stack" - the collection of parts of Exercism you
+# want to run locally. For now just copy the default file.
 cp stack.default.yml stack.yml
-# edit stack.yml to enabled the services (runners, analyzers, you'll be using)
-./bin/build_stack
 
-# Build everything in parallel
-docker-compose build --parallel
+# Start everything
+./bin/start
```

Comment:
Not cross-env; probably want to put a note in there for weird windows users (weird in the sense that they do have shell/shebang but use non-posix dirs). Maybe best to add "WSL2 recommended for windows, git bash works too". Source: on this machine I don't have WSL2, but I do have git bash.

Reply:
Could you give me a suggestion please (doesn't have to be a GH suggestion). I have no idea what needs to be said or how it should be worde.

Reply:
`./bin/start` is a _bash_ script, which won't work by default on Windows (for example using cmd.exe or PowerShell). However, it will work in linux and linux-simulated environments. Windows now supports 
[WSL, Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10), which allows you to run and use Linux directly on windows. These scripts work using WSL. 

Alternatively, [Git for Windows](https://github.com/git-for-windows/git/) comes with a _compatibility layer_ for linux, including `git bash`, known as MINGW64. We don't officially support it, but most likely everything will work there as well.